.babelrc

```diff
@@ -1,11 +1,49 @@
 {
-  "presets": [
-    ["es2015", { "modules": false }],
-    "react",
-    "stage-0"
-  ],
-  "plugins": [
-    "transform-es2015-modules-commonjs",
-    "dynamic-import-node"
-  ]
+    presets: [
+        ['@babel/preset-env', {
+            targets: {
+                chrome: 66,
+                firefox: 60,
+                edge: 41,
+                ie: 11,
+                ios: 11,
+            },
+            modules: false,
+            debug: false,
+            useBuiltIns: 'usage',
+            shippedProposals: true,
+        }],
+        ['@babel/preset-react', {
+            useBuiltIns: true,
+        }],
+    ],
+    plugins: [
+        '@babel/plugin-proposal-class-properties',
+        '@babel/plugin-syntax-dynamic-import',
+    ],
+    env: {
+        test: {
+            presets: [
+                ['@babel/preset-env', {
+                    targets: {
```

Comment:
The duplication here is unfortunate. Would it make sense to switch to a `.babelrc.js` and declare these targets or presets as a `const`?

Reply:
Ya. I don't like it but jest doesn't support anything but .babelrc. They claim to support .babelrc.js but for whatever reason it didn't work for me when I tried it.